### PR TITLE
https://github.com/stolostron/backlog/issues/26958

### PR DESCRIPTION
**Pull request by oafischer**
Still playing around with this, open to any suggestions.

### CODE REVIEW

File: clusters/cluster_lifecycle/assign_rbac_clusterset.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
+[#assign-role-managedclusterset]
+= Assigning RBAC permissions to a _ManagedClusterSet_
+
```
Comment (swopebe) [inline clusters/cluster_lifecycle/assign_rbac_clusterset.adoc:3]: hmmm, we have RBAC, did you check if this was over there?

File: clusters/cluster_lifecycle/clusterset_intro.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
+[#managedclustersets-intro]
+= Managing _ManagedClusterSets_
```
Comment (swopebe) [inline clusters/cluster_lifecycle/clusterset_intro.adoc]: This is an intro, so it should be titled as such, the intro is a concept (not task) that contains the tasks.

File: clusters/cluster_lifecycle/assign_rbac_clusterset.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/assign_rbac_clusterset.adoc:3]: Nothing about managedclustersets there at the moment.

File: clusters/cluster_lifecycle/create_clusterset.adoc [hunk 3]
```diff
@@ -0,0 +1,101 @@
+[#creating-a-managedclusterset]
+= Creating a ManagedClusterSet
```
Comment (swopebe) [inline clusters/cluster_lifecycle/create_clusterset.adoc]: we italicize in other titles

**Review by swopebe — APPROVED** [review]

File: clusters/cluster_lifecycle/create_clusterset.adoc (hunk 3 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/create_clusterset.adoc]: Good catch, missed that one!

**Review by cadawson — COMMENTED** [review]
Looks good. Thanks for doing that.

**Pull request merged**